protocols/v2/binary-sv2/no-serde-sv2/codec/src/datatypes/non_copy_data_types/seq_inner.rs

```diff
@@ -95,6 +95,13 @@ impl<'a, T: GetSize> GetSize for Seq0255<'a, T> {
     }
 }
 
+/// Fixed size data sequence up to a length of 65535
+/// 
+/// Byte Lenght: Fixed size T: `2 + LENGTH * size(T)Variable length T: 2 + seq.map(\|x\| x.length).sum()` 
```

Comment:
/// Byte Length Calculation:
/// - For fixed-size `T`: `2 + LENGTH * size_of::<T>()`
/// - For variable-length `T`: `2 + seq.map(|x| x.length()).sum()`

Reply:
ack. Good update. Thank you.